tools/rapids-github-run-id

```diff
@@ -13,8 +28,8 @@ case "${RAPIDS_BUILD_TYPE}" in
       --branch "${RAPIDS_REF_NAME}" \
       --commit "${RAPIDS_SHA}" \
       --workflow "${RAPIDS_BUILD_WORKFLOW_NAME:-pr.yaml}" \
-      --json databaseId \
-      --jq '.[0] | .databaseId')}
+      --json 'createdAt,databaseId' \
+      --jq 'sort_by(.createdAt) | reverse | .[0] | .databaseId')}
```

Comment:
See "How I tested this" in the PR description.

In short, `gh run list` does not make any strong guarantees about sorting, so just saying "choose the first element in the returned list" is not safe.

docs on `gh run list`: https://cli.github.com/manual/gh_run_list